docs/pages/es/overview.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: page
+title: Descripción General
+permalink: es/overview
+---
+# Introducción a Meshery
+El plano de gestión de la malla de servicios adopta, opera y desarrolla sobre diferentes mallas de servicios.
+Meshery facilita el aprendizaje sobre la funcionalidad y el desempeño de las mallas de servicio e incorpora la recopilación y visualización de métricas de las aplicaciones que se ejecutan en o entre las mallas de servicio.
+Meshery proporciona esta funcionalidad de alto nivel:
+
+1. Gestión del desempeño de la malla de servicios.
+1. Gestión de la configuración de la malla de servicios.
+     - Mejores prácticas de configuración.
+1. Gestión del ciclo de vida de la malla de servicios.
+1. Interoperabilidad y federación de malla de servicios.
+
+<iframe class="container" width="560" height="315" src="https://www.youtube.com/embed/CFj1O_uyhhs" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<div style="text-align:center;width:100%"><emphasis>Presentado en el Día de Servicio de Mallas 2019</emphasis></div>
+
+
+<h2>Qué desafíos resuelve Meshery?</h2>
+<b>Gestión de la Malla de Servicio - una o múltiples mallas de servicio.</b>
+
+Siempre que haya que responder a estas preguntas sobre el desempeño, son subjetivas a la carga de trabajo y la infraestructura específicas utilizadas para la medición. Ante este desafío, el proyecto Envoy, por ejemplo, se niega a publicar datos de desempeño porque tales pruebas pueden ser:
+- Enredadas
+- Malinterpretadas
+
+Más allá de la necesidad del desempeño y el alto consumo de datos bajo una permutación de diferentes cargas de trabajo (aplicaciones), los tipos y tamaños de los recursos de infraestructura, la necesidad de un proyecto multifuncional,
+ y las comparaciones entre semejantes, entre otros son anhelados para facilitar una comparación de las diferencias de comportamiento entre servicios de mallas y la selección de su uso. Los proyectos individuales son reservados en publicar resultados de pruebas de otras competencias de mallas de servicio. Es necesario un análisis independiente, imparcial y creíble.
+
+La intención de Meshery es ser un proveedor y una utilidad neutra de proyectos para evaluar de manera uniforme el desempeño de las mallas de servicio. Entre proyectos de malla de servicio y servicio de proxy (y sorprendentemente, dentro de un solo proyecto), existen diferentes números de herramientas y resultados. Meshery te permite elegir un conjunto eficiente de herramientas para su ecosistema al proporcionar evaluaciones y métricas de desempeño.
```

Comment:
// suggested changes in bold. 

La intención de Meshery es ser un proveedor y una utilidad neutra de proyectos para evaluar de manera uniforme el desempeño de las mallas de servicio. Entre proyectos de malla de servicio y servicio de proxy (y sorprendentemente, dentro de un **mismo** proyecto), existen diferentes números de herramientas y resultados. Mesher**y p**ermite elegir un conjunto eficiente de herramientas para su ecosistema al proporcionar evaluaciones y métricas de desempeño.